RPi-Pico/config/user_settings.h

```diff
@@ -33,8 +33,11 @@ extern "C"
 #endif
 
 #include <stdio.h>
+#include <time.h>
 #define TARGET_EMBEDDED
 
+#define WOLFSSL_RPIPICO
```

Comment:
Please add `WOLFSSL_IGNORE_FILE_WARN` to suppress the warning.

Reply:
Done